go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/dapr/go-sdk
 
-go 1.16
+go 1.15
```

Comment:
Just curious is this downgrade mandatory?

Reply:
It's a typo I think, i would fix soon.